packs/vsphere/actions/get_vm.json

```diff
@@ -0,0 +1,69 @@
+{
+    "name": "get_vm",
+    "runner_type": "run-python",
+    "description": "Retrieves the virtual machines on a vCenter Server system. It computes the union of Virtual Machine sets based on each parameter.",
```

Comment:
So as such we would want a mutually-exclusive group of parameters but st2 actions don't support mutually exclusive. Anyway, this works as the impl to return a union.